test/README-external.md

```diff
@@ -118,6 +118,35 @@ algorithms to be skipped, e.g. as follows:
 The names of all supported quantum-safe algorithms are available at
 <https://github.com/open-quantum-safe/oqs-provider#algorithms>
 
+pkcs11-provider test suite
+======================
+
+This builds and runs pkcs11-provider tests agains the local OpenSSL build.
+
+You will need a git checkout of pkcs11-provider at the top level:
+
+    $ git submodule update --init
+
+Then configure/build OpenSSL enabling external tests:
+
+    $ ./config shared enable-external-tests
+    $ make
+
+pkcs11-provider requires meson and cmake for the build process. Moreover, it 
```

Comment:
I don't see a trace of CMake files in https://github.com/latchset/pkcs11-provider.git

Reply:
Ah, you're right, I fixed that and rewrote the dependencies in distro-agnostic way too:

```
pkcs11-provider requires meson for the build process. Moreover, it requires
softhsm and nss softokn tokens and certtool, certutil, pkcs11-tool and expect
to run the tests.
```